### ISSUE TITLE
No way to quickly get back to top of page after clicking jump to section link

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/allsearch_frontend#488]
### Steps to recreate
1. Go to a [search results page on the allsearch](https://allsearch.princeton.edu/?q=potato)
2. Tab to the list of skip-to links (Catalog, Articles+, etc.)
3. Hit enter on a skip-to link
4. Note that there is no way to get back to the top of the page, to select a different skip-link, or search another term. No special key binding available either, and existing keybinding of F6 does not work to bring focus back to top of page.

See [fable user test at 01:09:30](https://app.makeitfable.com/share/67dd89292b3052f58338228b?video=1)

> So we made it down to the Journal section, that is powerful. That is a powerful accessibility tool. But I gotta say - hopefully there's a way to jump my focus back up. [...] Hopefully, there's a special key binding. I'm gonna hit F6 real quick. Jumps of to the URL, goes to the tab, comes back to where we were on the web page. That's not it. No indicators jumped up and said, 'Hey, this is a special key bind that could help you navigate.' So that's also out. [...] Not going through the learning process of learning how this functions, I'm assuming I have to tab-shift all the way back up.

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
